_posts/2023-02-02-board-updates.md

```diff
@@ -0,0 +1,31 @@
+---
+author_avatar: defna
+author_name: Jeff Triplett
+author_url: /about
+categories: ""
+date: 2023-02-17 06:00:00
+feature_image: 2017-07-20
```

Comment:
The last blog post had a value of `feature_image: 2023-01-01`. I'm not quite sure what this is for, do we need to update it?

Reply:
🤔 I don't think that one worked. https://www.defna.org/announcements/2023/01/01/call-for-proposals-for-djangocon-2023-website/

Reply:
Ugh, duh. I should have checked the rendered content. Good call.